### DISCUSSION TITLE
blog/como-crear-una-api-rest-con-nextjs-y-typescript

### DISCUSSION THREAD

**Discussion by giscus** [adrianRacovitaJSX/portfolio#3] [General]
# blog/como-crear-una-api-rest-con-nextjs-y-typescript

Aprende a construir una API REST robusta y tipada usando Next.js y TypeScript. Desde configuración básica hasta patrones avanzados y mejores prácticas.

http://localhost:3000/blog/como-crear-una-api-rest-con-nextjs-y-typescript

**Comment by adrianRacovitaJSX** [comment]
Interesante post!
